http-server-microservice-provider/src/main/java/io/silverware/microservices/providers/http/HttpServerMicroserviceProvider.java

```diff
@@ -131,6 +131,11 @@ public void run() {
             }
             this.server.start(builder);
             this.server.deploy(deploymentInfo());
+            log.info("Starting server at {}:{}/{}{} ",
+                  readProperty(HTTP_SERVER_ADDRESS),
+                  readProperty(HTTP_SERVER_PORT),
+                  readProperty(HTTP_SERVER_REST_SERVLET_MAPPING_PREFIX),
+                  readProperty(HTTP_SERVER_REST_CONTEXT_PATH));
             log.info("Started Http Server.");
```

Comment:
Since a new log info message has been added, this one can be removed. The new one should also be renamed from _Starting server at_ to _Started HTTP server at_ because by the time it is logged the server is already started, isn't it?